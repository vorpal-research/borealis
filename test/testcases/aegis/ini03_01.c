int main(void) {
    int *p;
    return *p;
}
