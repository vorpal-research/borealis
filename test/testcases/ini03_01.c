//
// Разыменование неинициализированного или нулевого указателя
//

int main(void)
{
    int *p;
    return *p; // дефект INI-03
}
