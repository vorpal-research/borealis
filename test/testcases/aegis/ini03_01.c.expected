
[
	
	{
		"location" : 
		{
			"filename" : "test/testcases/aegis/ini03_01.c",
			"loc" : 
			{
				"col" : 5,
				"line" : 3
			}
		},
		"type" : "NDF-01"
	}
]
